### PR TITLE
Upgrade eqt to v1.0.0

### PR DESCRIPTION
**Pull request by DanicaSTFC**
## Describe your changes
Updated eqt requirements to >=1.0.0
Updated yml recipe to fix environment

## Describe any testing you have performed
*Consider adding example code to [examples](https://github.com/vais-ral/CILViewer/tree/pr-template/Wrappers/Python/examples)*
- created a new environment with the yml file including eqt 1.00 and launched the standalone viewer. Tested with numpy and raw datasets
- ran the unit tests and all passed

## Link relevant issues
closes issue https://github.com/vais-ral/CILViewer/issues/378

## Checklist when you are ready to request a review

- [x] I have performed a self-review of my code
- [ ] I have added docstrings in line with the guidance in the [CIL developer guide](https://tomographicimaging.github.io/CIL/nightly/developer_guide.html)
- [ ] I have implemented unit tests that cover any new or modified functionality
- [x] CHANGELOG.md has been updated with any functionality change
- [x] Request review from all relevant developers
- [x] Change pull request label to 'waiting for review' 

## Contribution Notes
- [x] The content of this Pull Request (the Contribution) is intentionally submitted for inclusion in CILViewer (the Work) under the terms and conditions of the [Apache-2.0 License](https://spdx.org/licenses/Apache-2.0.html)
- [x] I confirm that the contribution does not violate any intellectual property rights of third parties

Qt contributions should follow Qt naming conventions i.e. camelCase method names.

VTK contributions should follow VTK naming conventions i.e. PascalCase method names.

### CODE REVIEW

File: docker/web-app/environment.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ dependencies:
   - pip
   - schema
   - pyyaml
-  - eqt>=0.7.0
+  - eqt>=0.7.1
```
Comment (paskino) [inline docker/web-app/environment.yml:11]: Have you tested this?

File: Wrappers/Python/conda-recipe/meta.yaml [hunk 2]
```diff
@@ -35,7 +35,7 @@ requirements:
     - numpy
     - vtk
     - pyside2
-    - eqt=0.7.1
```
Comment (paskino) [inline Wrappers/Python/conda-recipe/meta.yaml:38]: I think this should be explicitly request eqt version 1 as the change is backwards incompatible

**Review by paskino — CHANGES_REQUESTED** [review]

File: docker/web-app/environment.yml (hunk 1 above)
Comment (DanicaSTFC) [reply docker/web-app/environment.yml:11]: I did, see updated comment above

File: Wrappers/Python/conda-recipe/meta.yaml (hunk 2 above)
Comment (DanicaSTFC) [reply Wrappers/Python/conda-recipe/meta.yaml:38]: ok, changed

**Review by paskino — APPROVED** [review]

**Pull request merged**